### DISCUSSION TITLE
Downloading videos not working

### DISCUSSION THREAD

**Discussion by jaydenss** [dayanch96/BHTikTok-Plus#36] [General]
The last 2 days I haven’t been able to download videos (without watermark) it says it’s saved but nothing saves, tested on my iPhone 13 and iPhone 8, same issue on both, I’ve tried reinstalling, changing settings etc. saving images and videos with watermark work but saving without watermark/ HD it doesn’t seem to work atm.

**Comment by dyarfaradj** [comment]
Same problem here

**Comment by dayanch96** [comment]
Will be fixed in the next version. I’m working on update

**Reply by dyarfaradj** [reply]
Awesome! Also is it possible to make the download button download the HD version directly? Without getting the selection of choosing between HD or not?

**Reply by dayanch96** [reply]
Will see

**Reply by jaydenss** [reply]
Thank you for being so fast to reply! do you have a donation link? Thanks!

**Reply by XMRUSD** [reply]
Can you also add a feature to disable double tap to like. Sometimes I accidently double tap the screen when trying to hit the download button.
